ingest/api/messages.py

```diff
@@ -29,14 +33,14 @@ def build_message(file: [object], input_type: str, uuid_prefix: str, schema_path
         json_msg["id"] = message_uuid
         json_msg["properties"]["metadata_id"] = message_uuid
         json_msg["properties"]["data_id"] = message_uuid
-
         json_msg["properties"]["pubtime"] = datetime.now(timezone.utc).isoformat()
         try:
```

Comment:
Validation should no longer be needed with jsonschema, but handled by pydantic.

Reply:
We are making the metadata_id, data_id after receiving the actual data .so don't we need to verify whether these fields exist or not?